06-files-input-and-output/remove_files.py

```diff
@@ -1,49 +0,0 @@
-# Copyright (c) 2016 Ami . All rights reserved
```

Comment:
I think you deleted these files by accident. If I merge this branch to master they will be deleted there too.. Try and revert this commit.